torchci/lib/drciUtils.ts

```diff
@@ -40,7 +40,7 @@ export const EXCLUDED_FROM_FLAKINESS = [
   "pr-sanity-checks",
   // TODO (huydhn): Figure out a way to do flaky check accurately for build jobs
   "/ build",
-  "check-labels",
+  "Check labels",
```

Comment:
Instead of replacing the label, we should have both spellings mentioned here, so that users with older PR base are still covered..